src/ReactWrapper.jsx

```diff
@@ -915,3 +916,12 @@ export default class ReactWrapper {
     unmountComponentAtNode(this.options.attachTo);
   }
 }
+
+
+if (ITERATOR_SYMBOL) {
+  ReactWrapper.prototype[ITERATOR_SYMBOL] = function iterator() {
```

Comment:
i might be being over-pedantic here, but perhaps we should `Object.defineProperty` this so it's non-enumerable?